R/zzz.R

```diff
@@ -6,4 +6,19 @@ globalVariables(".")
 #'@importFrom magrittr "%>%"
 #'@importFrom rlang .data 
 #'
-NULL
\ No newline at end of file
+NULL
+
+startupmsg <- function() {
+"
+   / \\    Pandora contains many samples that are ethically
+  / | \\   and/or culturally sensitive and are therefore
+ /  *  \\  off-limits for many analyses.
+
+They are indicated in the column Ethically_culturally_sensitive of TAB_Sample. Make sure to exclude these samples depending on your application and permissions!"
```

Comment:
```suggestion
They are indicated in the column Ethically_culturally_sensitive of TAB_Sample. 

Ensure to exclude these samples and all upstream and downstream entries depending on permissions and context!

The sidora devs are not liable for any problems for any ethical problems that may arise from the use of this software.
"
```

Reply:
Ah you can't suggest on a comment- so I've updated for you. thanks for fixing my early morning mistakes :laughing: